src/scss/root/_index.scss

```diff
@@ -39,6 +53,58 @@ $tokens: () !default;
 	}
 }
 
+//
+// token-map-to-css-variable
+// mixin to output CSS Variable from a token map
+//
+// $tokens - A map of tokens key values
+// $name - The name of the sub map in which to output from
+// $prefix - The prefix value to add to the CSS Variable name
+//
+@mixin token-map-to-css-variable($tokens, $name, $prefix: '') {
```

Comment:
Maybe token prefix should default to `--t-componentName-property`?  If someone leaves off the prefix, then it would be `---componentName-property`.

Reply:
Updated